orcid-web/src/main/resources/freemarker/includes/ng2_templates/link-account-ng2-template.ftl

```diff
@@ -24,7 +24,7 @@
     <div *ngIf="shibbolethSignin || socialSignin"> 
         <div *ngIf="signinData && signinData.unsupportedInstitution" class="col-md-9 col-sm-9 col-sm-push-3 col-md-push-3">
             <p class="alert">${springMacroRequestContext.getMessage("social.link.unsupported.unsuccessful.1")}<span *ngIf="signinData.accountId">${springMacroRequestContext.getMessage("social.link.unsupported.unsuccessful.2")}{{signinData.accountId}}${springMacroRequestContext.getMessage("social.link.unsupported.unsuccessful.3")}</span>${springMacroRequestContext.getMessage("social.link.unsupported.unsuccessful.4")}</p>
-            <p>${springMacroRequestContext.getMessage("social.link.unsupported.we_have_made_a_note.1")}<a href="mailto:{{signinData.institutionContactEmail}}?cc=support@orcid.org&amp;subject=${springMacroRequestContext.getMessage("social.link.unsupported.email.subject")?url}&amp;body=${springMacroRequestContext.getMessage("social.link.unsupported.email.body")?url}">${springMacroRequestContext.getMessage("social.link.unsupported.we_have_made_a_note.2")}</a>${springMacroRequestContext.getMessage("social.link.unsupported.we_have_made_a_note.3")}<a href="{{getBaseUri()}}/signin">${springMacroRequestContext.getMessage("social.link.unsupported.we_have_made_a_note.4")}</a>${springMacroRequestContext.getMessage("social.link.unsupported.we_have_made_a_note.5")}</p>
```

Comment:
@lizkrznarich Just removed the CC to support@orcid.org, do we want to keep it?

Reply:
@leomendoza123 I think this is the one exception where we should keep support@orcid.org bc we do want users to copy us when they contact their IdP admin about institutional signin.